bowler/main.py

```diff
@@ -27,7 +27,20 @@ def main(ctx: click.Context, debug: bool, version: bool) -> None:
 
     if debug:
         BowlerTool.NUM_PROCESSES = 1
-    logging.basicConfig(level=(logging.DEBUG if debug else logging.WARNING))
+
+    root = logging.getLogger()
+    if not root.hasHandlers():
```

Comment:
Is this to allow tests to already set up a logger?  I'm curious what is this defensive against.